app/views/return-requirements/check.njk

```diff
@@ -27,81 +34,81 @@
         {
           classes: 'govuk-summary-list govuk-summary-list__row--no-border',
           key: {
-            text: "Start date"
-          },
+          text: "Start date"
+        },
           value: {
-            html: '<span data-test="meta-data-start-date">' + startDate + '</span>'
-          },
+          html: '<span data-test="meta-data-start-date">' + startDate + '</span>'
+        },
           actions: {
-            items: [
-              {
-                href: "/system/return-requirements/" + sessionId + "/start-date",
-                text: "Change",
-                visuallyHiddenText: "the start date for the return requirement"
-              }
-            ]
-          }
+          items: [
+            {
+              href: "/system/return-requirements/" + sessionId + "/start-date",
+              text: "Change",
+              visuallyHiddenText: "the start date for the return requirement"
+            }
+          ]
+        }
         },
         {
           classes: 'govuk-summary-list govuk-summary-list__row--no-border',
           key: {
-            text: "Reason"
-          },
+          text: "Reason"
+        },
           value: {
-            html: '<span data-test="meta-data-reason">' + reason + '</span>'
-          },
+          html: '<span data-test="meta-data-reason">' + reason + '</span>'
+        },
           actions: {
-            items: [
-              {
-                href: reasonLink,
-                text: "Change",
-                visuallyHiddenText: "the reason for the return requirement"
-              }
-            ]
-          }
+          items: [
+            {
+              href: reasonLink,
+              text: "Change",
+              visuallyHiddenText: "the reason for the return requirement"
+            }
+          ]
+        }
```

Comment:
A bunch of indentations seem to have been removed that I think were valid. Was this intentional or some auto-formatter?